Forms/MainWindow.cpp

```diff
@@ -125,7 +125,7 @@ void MainWindow::setupModels()
     languageGroup->setExclusive(true);
     connect(languageGroup, &QActionGroup::triggered, this, &MainWindow::slotLanguageChanged);
     currentLanguage = setting.value("settings/locale", "en").toString();
-    QStringList locales = { "de", "en", "es", "fr", "it", "ja", "ko", "zh" };
+    QStringList locales = { "de", "en", "es", "fr", "it", "ja", "ko", "zh", "tw" };
```

Comment:
This is causing a runtime crash. There needs to be another QAction added to the QMenu for language for the added language. See the image below to where I am referencing.


![image](https://user-images.githubusercontent.com/24730718/72020529-3441d900-3229-11ea-89fb-5ae18d72ba83.PNG)

Reply:
Hi Owner, thanks for pointing out the problem.
Actually, I am not familiar with QT and even have not setup the compile environment yet.
Just my friends really need a tw(CHT) Language version RaidFinder, so I tried to translate it in tw(CHT).
Unfortunately, it's not easy as modify string files only and then cause crash. Sorry. 
Now, I added all of tw(CHT) string files and committed them. 
Could you please help me to add code of QAction/QMenu and finish the feature? Thanks a lot.

Wayne

Reply:
Yeah I can take care of the rest.